rust/src/core.rs

```diff
@@ -66,7 +66,7 @@ macro_rules!BIT_U64 {
 //
 // Function types for calls into C.
 //
-pub enum StreamingBufferConfig {}
+pub use suricata_sys::sys::StreamingBufferConfig;
```

Comment:
smaller diff than importing `suricata_sys::sys::StreamingBufferConfig;` instead of `core::StreamingBufferConfig`

Reply:
nice, will add